decNumber/CMakeLists.txt

```
@@ -31,6 +31,8 @@ target_include_directories(decNumber
 
 set_target_properties(decNumber
     PROPERTIES
+        VERSION ${PROJECT_VERSION}
```

We're using version 3.68 of decNumber, I'd prefer if we used that rather than the ion-c project version.
```suggestion
        VERSION "3.68"
```